install/0000_50_machine-config-operator_05_osimageurl.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: machine-config-osimageurl
+  namespace: openshift-machine-config-operator
+data:
+  # This is a dummy value
```

Comment:
Minor: some more info here would be nice. E.g. "(it will be replaced by the associated osImageUrl for a given release payload)".